src/scc_hypervisor_collector/api/uploader.py

```diff
@@ -5,7 +5,59 @@
 collected from the specified backends to the SCC using the provided
 credentials.
 """
+import json
```

Comment:
Surprised that pylint didn't complain about the ordering or whitespace separation of these imports...